src/js/server/devpmt-express.js

```diff
@@ -0,0 +1,528 @@
+/**
+ * Devpmt Express Server
+ *
+ * Contains `gpii.devpmt` the primary component for our express server,
+ * and other tooling it requires.
+ *
+ * Copyright 2019 Raising the Floor - International
+ *
+ * Licensed under the New BSD license. You may not use this file except in
+ * compliance with this License.
+ *
+ * You may obtain a copy of the License at
+ * https://github.com/GPII/universal/blob/master/LICENSE.txt
+ */
+/* eslint-env node */
+"use strict";
+
+var session = require("express-session");
+var RedisStore = require("connect-redis")(session);
+
+var fluid = require("infusion");
+var gpii = fluid.registerNamespace("gpii");
+fluid.require("gpii-express");
+fluid.require("gpii-express-user");
+fluid.require("gpii-handlebars");
+require("gpii-universal");
+
+fluid.registerNamespace("gpii.devpmt");
+fluid.registerNamespace("gpii.handlebars");
+
+
+/**
+ * gpii.handlebars.requestFuncTransform
+ *  This component is sort of a facsimile of the functions available
+ *  in many web frameworks that take a request instance or URL parameter
+ *  and return something for the response. In this case what is returned
+ *  will be assigned to the variable in the transform.
+ */
+fluid.defaults("gpii.handlebars.requestFuncTransform", {
+    gradeNames: ["fluid.component"]
+});
+
+gpii.handlebars.requestFuncTransform = function (value, transformSpec) {
+    return fluid.invokeGlobalFunction(transformSpec.func, [transformSpec.that, value]);
+};
+
+/**
+ * Allows fetching a component from the tree using an IoC expression of the
+ * type one would normally place in a components defaults block. If there is
+ * more than one instance of this component, will use the first instance in
+ * the returned list.
+ *
+ * An example path might be:
+ * "{gpii.devpmt}.options.prefSetDir"
+ *
+ * but could be any resolvable path on the component.
+ *
+ * @param {String} path - The IoC path to look up.
+ * @return {Any} The value at the path.
+ */
+fluid.getValueByGlobalPath = function (path) {
```

Comment:
Looks like this function is disused as well

Reply:
this is removed